.github/workflows/ci-tlv-tool.yml

```diff
@@ -0,0 +1,36 @@
+name: CITLVTool
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: "20 7 * * *"
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build_tlv_tool:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fmt
+        run: cd tools/tlv; cargo fmt -- --check
```

Comment:
[nits] I suggest using `working-directory`.

Reply:
Not sure how that would help. Would you mind pasting a code snippet?

Reply:
```yaml
        run: cargo fmt -- --check
        working-directory: tools/tlv
```

Reply:
Oh, now I get it. Will do - thanks!